epub33/core/biblio.js

```diff
@@ -3,6 +3,18 @@ var biblio = {
 		"title": "CSS Snapshot",
 		"href": "https://www.w3.org/TR/CSS/"
 	},
+	"DPUB-ARIA" : {
+		"authors": [
+			"Matt Garrish",
+			"Tzviya Siegman",
+			"Markus Gylling",
+			"Shane McCarron"
+		],
+		"title": "Digital Publishing WAI-ARIA Module 1.0",
+		"href": "https://www.w3.org/TR/dpub-aria/",
+		"date": "14 December 2017",
+		"publisher": "W3C"
+	},
```

Comment:
For "latest" reference we've typically dropped the editors, date and version number. Should we do similar here?
```suggestion
	"DPUB-ARIA" : {
		"title": "Digital Publishing WAI-ARIA Module",
		"href": "https://www.w3.org/TR/dpub-aria/",
		"publisher": "W3C"
	},
```